src/o365/spo/commands/list/list-webhook-list.ts

```diff
@@ -124,15 +142,23 @@ class SpoListWebhookListCommand extends SpoCommand {
     const options: CommandOption[] = [
       {
         option: '-u, --webUrl <webUrl>',
-        description: 'URL of the site where the list to retrieve webhooks for is located'
+        description: `URL of the site where the list to retrieve webhooks for is located`
+      },
+      {
+        option: '--listId [listId]',
+        description: `ID of the list to retrieve all webhooks for. Specify either id or title but not both`
+      },
+      {
+        option: '--listTitle [listTitle]',
+        description: `Title of the list to retrieve all webhooks for. Specify either id or title but not both`
       },
       {
         option: '-i, --id [id]',
```

Comment:
The shorthand `-i` should point to the new `listId` option so that people using it won't be getting deprecation warnings